packages/jest-config/src/ValidConfig.ts

```diff
@@ -110,7 +111,10 @@ const initialOptions: Config.InitialOptions = {
   skipFilter: false,
   skipNodeResolution: false,
   slowTestThreshold: 5,
-  snapshotFormat: PRETTY_FORMAT_DEFAULTS,
+  snapshotFormat: {
+    ...PRETTY_FORMAT_DEFAULTS,
+    compareKeys: () => {},
+  } as SnapshotFormat,
```

Comment:
Am I proud of this fix? No.
Does it work? Yes.

I'm afraid that I won't have time to work on the improved fix though - this is somewhat problematic because this thing is handled through a very generic validation function and this case here is very specific.

If you have any suggestions on the improved design that could be implemented **quickly** I could take a stab at this but I'm worried that an improved architecture around this might not be a change for a couple of lines of code.

Reply:
Probably the long-run plan is to use recently added schemas for validation purposes? This would IMHO go outside of the scope of this PR as it's a bigger refactor.

Reply:
>Probably the long-run plan is to use recently added schemas for validation purposes?

Correct 🙂